mirrulations-work-server/src/mirrserver/work_server.py

```diff
@@ -95,6 +95,30 @@ def put_results(workserver, data):
     return (True,)
 
 
+@check_for_database
+def post_attachments(workserver, data):
+    client_id = request.args.get('client_id')
+    success, *values = check_request_had_valid_client_id(workserver, client_id)
+    if not success:
+        return False, values[0], values[1]
+    if 'error' in data['results'] or 'errors' in data['results']:
+        job_id = data['job_id']
+        result = workserver.redis.hget('jobs_in_progress', job_id)
+        workserver.redis.hdel('jobs_in_progress', job_id)
+        workserver.redis.hset('invalid_jobs', job_id, result)
+        return (True,)
+    success, *results = check_results(workserver, data, int(client_id))
+    if not success:
+        return (success, *results)
+    job_id = data.get('job_id')
+    workserver.redis.hdel('jobs_in_progress', job_id)
+    # Code for getting attachment data and saving to mongo goes here
+    # Save data to mongo
```

Comment:
The endpoint for handling attachments is extremely similar to the endpoint for putting data for now. These lines are where we will need to make changes for handling the attachments and saving them in a particular way to mongo.